testsuite/tests/typing-poly/poly.ml

```diff
@@ -520,7 +520,7 @@ end
 Line 3, characters 12-17:
 3 |   method id x = x
                 ^^^^^
-Error: This method has type "'a -> 'a" which is less general than "'b. 'b -> 'a"
+Error: This method has type "'b -> 'b" which is less general than "'c. 'c -> 'b"
```

Comment:
Here also, the error message names end up being farther away from the source names.